_layouts/home.html

```diff
@@ -21,10 +31,33 @@ <h1 class="page-heading">{{ page.title }}</h1>
     {%- if page.list_title -%}
       <h2 class="post-list-heading">{{ page.list_title }}</h2>
     {%- endif -%}
-    <ul class="post-list">
+    <!-- <ul class="post-list"> -->
       {%- assign date_format = site.minima.date_format | default: "%b %-d, %Y" -%}
       {%- for post in posts -%}
-      <li>
+        {% assign tags = post.tags %}
+        <div class="row">
+          <div class="card mb-3">
+            <div class="card-header">
+              {{ post.author }}
+            </div>
+            <div class="card-body">
+              <h5 class="card-title">
+                <a href="{{ post.url | relative_url }}">{{ post.title | escape }}</a>
+              </h5>
+              {%- if site.show_excerpts -%}
+                <p class="card-text">{{ post.excerpt }}</p>
+              {%- endif -%}
+              Tags: 
+              {%- for tag in tags -%}
+                <a href="#" class="badge badge-primary ml-2">{{ tag }}</a>
```

Comment:
Is it easy to make these links actually do something? e.g. Render a page with all the posts with the same tag?

Reply:
I think this can be done with a jekyll plugin, but gh-pages doesn't allow plugins. We could generate the tag pages locally and then push them to the repo or we could use [this non-plugin way](https://stackoverflow.com/questions/1408824/an-easy-way-to-support-tags-in-a-jekyll-blog/21002505#21002505) - autogenerating a list of all tags and posts associated with each tag. 

I would favour the second option I think - unless I'm missing something, which is highly likely.

Reply:
Addressed second option in below commit.